### PR TITLE
Update Client Examples Kafka version to 3.4.0

### PR DESCRIPTION
**Pull request by OwenCorrigan76**
Update Kafka dependencies to version 3.4.0, in keeping with current versions in The Bridge and The Operator.
Also update openTelemetry versions to 1.19.0

### CODE REVIEW

File: java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java [hunk 1]
```diff
@@ -196,11 +196,11 @@ private Future<List<ConsumerRecord>> poll() {
...
-                                json.getString("topic"),
-                                json.getValue("key"),
-                                json.getValue("value"),
-                                json.getInteger("partition"),
-                                json.getLong("offset"))
+                                    json.getString("topic"),
+                                    json.getValue("key"),
+                                    json.getValue("value"),
+                                    json.getInteger("partition"),
+                                    json.getLong("offset"))
```
Comment (ppatierno) [inline java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java]: I guess this change is unrelated to the PR, can we avoid it please?

Comment (OwenCorrigan76) [reply java/http/vertx/java-http-vertx-consumer/src/main/java/HttpKafkaConsumer.java]: @ppatierno I carried some old commits from a rebase. Fixing now

**Review by ppatierno — APPROVED** [review]

**Review by scholzj — COMMENTED** [review]
@OwenCorrigan76 Did you test that it works, including the tracing?

**Comment by OwenCorrigan76** [comment]
@scholzj Yes, Jaeger and OpenTelemetry both tested and working in the Java/Kafka examples, as are the Http examples.

**Review by scholzj — APPROVED** [review]

**Comment by scholzj** [comment]
Thanks @OwenCorrigan76
